src/components/DetailsDriftAnalysis/detailsDriftAnalysis.util.js

```diff
@@ -0,0 +1,56 @@
+import { roundFloats } from '../../utils/roundFloats'
+
+export const generateDriftAnalysis = selectedItem => {
+  const measures = selectedItem.status.drift_measures
```

Comment:
If `drift_measures` is missing or `null`, we better default to an empty object so later when using the `.` operator on `measures` (for example, `measures.tvd_sum`) it will not result in an error

```suggestion
  const measures = selectedItem.status.drift_measures ?? {}
```